### DISCUSSION TITLE
ComboBox PointerWheelChanged not firing

### DISCUSSION THREAD

**Discussion by WoistdasNiveau** [AvaloniaUI/Avalonia#17134] [Q&A]
### Describe the bug

When using the Combobox and using the PointerWheelChanged event, is does not trigger when one tries to scroll.

### To Reproduce

Create a Combobox, create a method to listen for the PointerWheelChanged event

### Expected behavior

Should fire the same way as it does for the Datagrid to be able to load Usernames to select, for example dynamically.

### Avalonia version

11.1.3

### OS

Windows

### Additional context

I wanted to use the Combobox to select a user responsible for a task and as i did not want to load all users at once i wanted to implement infinite scroll via the PointerWheelChanged event.

**Comment by stevemonaco** [comment]
Have you tried a tunneling event? Controls themselves will typically have the first bite of a bubbling event and potentially handle it. If you're specifying the event handler in XAML, it's always a bubbling event. You need to manually AddHandler to create tunneling events.

**Reply by WoistdasNiveau** [reply]
Can you eloborate please an where i have to AddHandler?

**Reply by WoistdasNiveau** [reply]
I guess fro mthe documentation i would have to set Combobox.AddHandler(PointerWheelChanged, MethdToCall, RoutingStrategies.Tunnel). Is this correct? The ComboBox, however, lies within a DataGrid so i cannot just give it a Name and add the handler in the Code behind as it is gerneated dynamically with the data. Is there any other option then rewritng the Datagrid completely in C#  in the code behind?

**Reply by timunie** [reply]
You can listen to Loaded event of the ComboBox and inside this event, you know that `sender` is your control, where you can add the Tunnel event. How this is clear enough?

**Comment by timunie** [comment]
@WoistdasNiveau maybe you can also try AutoCompleteBox. It has an option for loading items async and user can start typing any character to search for users.

**Reply by WoistdasNiveau** [reply]
AutoCompleteBox would be a good alternative too, a mix of the Combo and AutoCompletebox would be perfect. The problem, however, i need to use both in a Dartagrid, so i cannot assign the AsyncPopulator like it is done in the avalonia examples and i could not figure out the correct way to bind the method from the Viewmodel. Can you tell me how to bind the ASyncPopulator from the ViewModel? I have tried creating a Property of Type Func<,> but nothing gets triggered.
